guides/source/configuring.md

```diff
@@ -2014,7 +2014,7 @@ Accepts a logger conforming to the interface of Log4r or the default Ruby Logger
 
 #### `config.action_view.erb_trim_mode`
 
-Gives the trim mode to be used by ERB. It defaults to `'-'`, which turns on trimming of tail spaces and newline when using `<%= -%>` or `<%= =%>`. See the [Erubis documentation](http://www.kuwata-lab.com/erubis/users-guide.06.html#topics-trimspaces) for more information.
```

Comment:
@Earlopain @byroot This reminded me of #49640 (someone could probably merge that), but in particular we could probably just link to the [rdoc](https://docs.ruby-lang.org/en/3.2/ERB.html#method-c-new) for this. Bonus points for upstreaming a doc PR to make this more apparent in the class documentation where it could even have a heading like [here](https://docs.ruby-lang.org/en/3.2/ERB.html#class-ERB-label-Notes).